### DISCUSSION TITLE
can not upload a file via git bash

### DISCUSSION THREAD

**Discussion by bloodsravenx** [community/community#51793] [Repositories] [answered]
### Select Topic Area

Show & Tell

### Body

i can not upload file in a git bash, any solutions?

**Comment by hellookii** [comment, marked as answer]
First create a new repository, then input other optionally, press add readme, then create repository, then press point 3 on the right, then upload files via computer without gitbash. good luck!

**Reply by bloodsravenx** [reply]
thanks a lot , very usefull!!
